polar-core/src/error.rs

```diff
@@ -475,6 +475,12 @@ pub enum OperationalError {
     InvalidState { msg: String },
     /// Serialization errors in the `polar-c-api` crate.
     Serialization { msg: String },
+    // This should go away once we can constrain the value variant of a particular term in the type
+    // system, e.g., `Term<String>` instead of `Term::value().as_string()`.
+    UnexpectedValue {
+        expected: String,
```

Comment:
Could make this `&'static str` for no runtime allocations.

Reply:
714256235